test/rm.js

```diff
@@ -308,3 +319,14 @@ test('remove fifo', t => {
     t.is(result.code, 0);
   });
 });
+
+test('remove symbolic link', t => {
```

Comment:
nit: `remove symbolic link to file`

Does this actually require `skipOnWin`? I would be surprised if this is so, because `remove symbolic link to a dir` passes without that.

Also, it might be nice to move this up closer to that test case.